webpack.common.js

```diff
@@ -126,19 +130,24 @@ module.exports = (env = {}) => ({
         use: {
           loader: 'file-loader',
           options: {
-            prefix: 'font/'
+            outputPath: 'font',
+            esModule: false
           }
         }
       },
       {
-        test: /\.(jpg|png|svg|ico)$/,
-        use: 'file-loader'
+        test: /\.(jpg|jpeg|png|svg|ico)$/i,
+        type: 'asset'
       }
     ]
   },
   plugins: [
     new MiniCssExtractPlugin({
       filename: '[chunkhash].css'
+    }),
+    new ESLintPlugin({fix: true, extensions: ['js', 'jsx']}),
+    new webpack.DefinePlugin({
+      'process.env.RESET_APP_DATA_TIMER': JSON.stringify('') // fix for algoliasearch
```

Comment:
please add a link to the related github issue/stack overflow thread or similar for clarity to the comment